Common/GPU/OpenGL/GLQueueRunner.cpp

```diff
@@ -817,6 +821,9 @@ void GLQueueRunner::PerformRenderPass(const GLRStep &step, bool first, bool last
 	CHECK_GL_ERROR_IF_DEBUG();
 	auto &commands = step.commands;
 	for (const auto &c : commands) {
+		if (IsVRBuild()) {
```

Comment:
This is a bit of a hot loop, it might make more sense to have `const bool isVR = IsVRBuild();` outside the loop.  If it's inline, it should still compile out, and if it's not, it might avoid needless calls.

-[Unknown]